src/qmflows/common.py

```diff
@@ -1,51 +1,49 @@
-"""A module with named tuples (:class:`~typing.NamedTuple`)."""
-
+"""common (:class:`~typing.NamedTuple`) used mostly for parsing."""
 from typing import Any, Type, NamedTuple, Callable, Optional, TYPE_CHECKING
-
+import numpy as np
 if TYPE_CHECKING:
     from pyparsing import ParserElement
 else:
     ParserElement = 'pyparsing.ParserElement'
 
 __all__ = ['AtomBasisKey', 'AtomBasisData', 'AtomXYZ', 'CGF',
-           'InfoMO', 'InputKey', 'MO', 'MO_metadata', 'ParseWarning']
+           'InfoMO', 'MO_metadata', 'ParseWarning']
 
 
 class AtomBasisKey(NamedTuple):
-    atom: Any
-    basis: Any
-    basisFormat: Any
+    """Namedtuple containing the `basisFormat` for a given `basis` and `atom`."""
+
+    atom: str
+    basis: str
+    basisFormat: list
 
 
 class AtomBasisData(NamedTuple):
-    exponents: Any
-    coefficients: Any
+    """Contraction coefficients and exponents for a gaussian basis."""
 
+    exponents: np.array
+    coefficients: np.array
```

Comment:
The array class is called ``np.ndarray``; ``np.array`` is a helper function for initializing the instances.

Reply:
thanks!